### PR TITLE
updates for 0.12

### PR DESCRIPTION
**Pull request by justinwoo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,13 @@ sudo: required
 language: node_js
 node_js: 10
 
+env:
+  - PATH=$HOME/purescript:$PATH
 install:
+  - TAG=$(wget -q -O - https://github.com/purescript/purescript/releases/latest --server-response --max-redirect 0 2>&1 | sed -n -e 's/.*Location:.*tag\///p')
+  - wget -O $HOME/purescript.tar.gz https://github.com/purescript/purescript/releases/download/$TAG/linux64.tar.gz
+  - tar -xvf $HOME/purescript.tar.gz -C $HOME/
+  - chmod a+x $HOME/purescript
```
Comment (spicydonuts) [inline .travis.yml]: Is this necessary now that 0.12 is on npm?

Comment (justinwoo) [reply .travis.yml]: I guess not, but it's hell of a lot faster to install

Comment (spicydonuts) [reply .travis.yml]: I'm more worried about my mental capacity. :wink:
I'll probably leave it in if no one else cares.

**Comment by xcthulhu** [comment]
I am super excited for this!

This resolves https://github.com/lumihq/purescript-react-basic/issues/9 as per @paf31 's suggestion.

File: .travis.yml (hunk 1 above)
Comment (paf31) [reply .travis.yml]: Personally, I prefer the current version since it pins the compiler version.

**Comment by paf31** [comment]
Thanks @justinwoo!

**Pull request merged**